### PR TITLE
docs(guides): user metadata

### PR DESCRIPTION
**Pull request by mffap**
Adds a guide how to manually request user medata as reserved scope from ZITADEL.

Depends on zitadel-tools https://github.com/zitadel/zitadel-tools/pull/44

### CODE REVIEW

File: docs/docs/guides/customization/user-metadata.md [hunk 1]
```diff
@@ -57,7 +57,29 @@ export BASIC_AUTH="$(go run basicauth.go -id $CLIENT_ID -secret $CLIENT_SECRET)"
 
 </TabItem>
 
-<TabItem value="other" label="Other">
+<TabItem value="python" label="Python" default>
+
+Grab zitadel-tools to create the [required string](https://docs.zitadel.com/docs/apis/openidoauth/authn-methods#client-secret-basic) for Basic authentication:
+
```
Comment (eliobischof) [inline docs/docs/guides/customization/user-metadata.md:63]:
```suggestion
```

File: docs/docs/guides/customization/user-metadata.md [hunk 2]
```diff
@@ -57,7 +57,29 @@ export BASIC_AUTH="$(go run basicauth.go -id $CLIENT_ID -secret $CLIENT_SECRET)"
 
 </TabItem>
 
-<TabItem value="other" label="Other">
+<TabItem value="python" label="Python" default>
```
Comment (eliobischof) [inline docs/docs/guides/customization/user-metadata.md:60]:
```suggestion
<TabItem value="python" label="Python">
```

File: docs/docs/guides/customization/user-metadata.md (hunk 1 above)
Comment (eliobischof) [reply docs/docs/guides/customization/user-metadata.md:63]: only for the go tab

**Review by eliobischof — CHANGES_REQUESTED** [review]

File: docs/docs/guides/customization/user-metadata.md [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+print(base64_message)
+```
+
+Export the result to the environment variable `BASIC_AUTH`.
+
+</TabItem>
+
+<TabItem value="js" label="Javascript" default>
+
+Grab zitadel-tools to create the [required string](https://docs.zitadel.com/docs/apis/openidoauth/authn-methods#client-secret-basic) for Basic authentication:
```
Comment (eliobischof) [inline docs/docs/guides/customization/user-metadata.md:84]:
```suggestion
```

**Comment by fforootd** [comment]
@eliobischof can you conclude the review here please. TY

**Review by eliobischof — APPROVED** [review]

**Pull request merged**